### ISSUE TITLE
Show error when agent is undefined

### ISSUE THREAD

**Issue by JobEijdems** [atomicdata-dev/atomic-data-browser#176]

**Comment by joepio** [comment]
Agreed that this error should be handled in some better way, but how did this state occur? Was it through UI interaction or through debugging / hard-setting values in devtools? Do you perhaps have reproduction steps?

**Comment by JobEijdems** [comment]
I'm not sure how this happened exactly, but it wasn't through devtools. If it happens again I can look for reproduction steps.
